charts/microgateway/README.md.gotmpl

```diff
@@ -1,153 +1,365 @@
+### Echo-Server
+
+For the first deployment it could be very useful to have a backend service processing requests. For this purpose the dependend echo-server can be deployed by doing the following:
+  custom-values.yaml
+  ```
+  echo-server:
+    enabled: true
+  ```
+
+Finally, apply the Helm chart configuration file with `-f` parameter.
 
 ```console
-helm uninstall waf
+helm upgrade -i microgateway airlock/microgateway -f custom-values.yaml
 ```
 
-## <a name="dsl-configuration"></a> DSL Configuration
-With the Helm Chart you have three different possibilities to configure the DSL of the Microgateway. 
-Depending on the environment and use-case, another option may be the best and easiest choice for the implementation. 
+:information_source: **Possible settings**:<br>
+Please refer to the [Echo-Server Helm chart](https://ealenn.github.io/Echo-Server/pages/helm.html) to see all possible parameters of the Echo-Server Helm chart.
+
+## DSL configuration
+
+The Helm chart provides three different possibilities to configure the Microgateway DSL.
+Depending on the environment and use-case, another option might suit better and is easier to implement it.
+
+### Simple DSL configuration
+
+The simple DSL configuration suites best for the following use case:
+
+Use Case)
+* Virtual Host 1: abc.com -> Mapping 1: / -> Backend Service 1
+
+This means that:
+* Only one Virtual Host is configured.
+* The Mapping configuration is applied to all requests sent to the backend service.
+* There is only one backend service.
+
+
 
-### Default DSL
 The Helm chart provides a simple configuration which can be configured with `config.default.*` parameters.
 All settings have already configured a default value. So only the values which differ from the default value have to be configured. 
 
-Example code fragment:
-Customization of the backend hostname.
+
+**Example:**
+
 custom-values.yaml
 ```
 config:
+  IPHeader:
+    trustedProxies:
+      - 10.0.0.0/28
   default:
+    mapping:
+      entryPath: /
     backend:
       hostname: custom-backend
+
+redis:
+  enabled: true
 ```
 
-### Custom DSL App
-If the default app settings are not sufficient, you can define a custom app as YAML with the `config.apps` parameter. 
-This setting overwrites the default app (mapping & backend), but the remaining settings of the DSL can still be configured with the default DSL method. 
-Example code fragment:
-Customization of the VirtualHost with Apache Expert Settings.
+**Parameters which can be used**:
+* All `config.*` parameters can be used.
+  * When using the `config.redisService` parameter:
+    * The Mapping will be configured with "enforce_session", except `config.default.mapping.sessionHandling` is configured.
+  * When using the `redis.enabled` parameter:
+    * The Mapping will be configured with "enforce_session", except `config.default.mapping.sessionHandling` is configured.
+* All `config.default.*` parameters can be used.
```

Comment:
Redundant to * All `config.*` parameters can be used.

Reply:
Yes, it is redundant but it explains that `config.default.mapping.sessionHandling` wins. 

Shall I remove line 204-207?

Reply:
I would remove line 208. 
The line 203 already includes the config.default.* settings. 
I think 204-207 are necessary because they describe a special setting.

Reply:
done